### ISSUE TITLE
Unable to access gptr.dev

### ISSUE THREAD

**Issue by JinayJain** [assafelovic/gpt-researcher#723]
I'm unable to access any pages on https://gptr.dev/

Chrome reports the following error:
![image](https://github.com/user-attachments/assets/b074bd86-01dc-4f90-be89-43eead482ddd)

I have also tried using Firefox which doesn't work either.

**Comment by danieldekay** [comment]
It works on my end. However at work, it is blocked by our VPN, as it is wrongly classified as parked domain.

**Comment by assafelovic** [comment]
Very strange @danieldekay and thanks for raising it. Works on my end. Have you tried entering it from another network like your mobile (not wifi)?
